monkestation/code/modules/floorsigns/floorsigns.dm

```diff
@@ -0,0 +1,76 @@
+/obj/effect/turf_decal/floorsign
+	icon = 'monkestation/code/modules/floorsigns.dmi'
```

Comment:
no. icons go in the `icons` folder - or, well, `monkestation/icons` in this case. also you put `floorsigns.dmi` instead of `floorsign_decals.dmi`.